.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,8 @@
 blank_issues_enabled: false
+contact_links:
+  - name: 💬 IRC
+    url: https://webchat.freenode.net/#newpipe
+    about: Chat with us via IRC for quick Q/A here
+  - name: 💬 Matrix
+    url: https://matrix.to/#/#newpipe:matrix.org
```

Comment:
Shouldn't this be `#freenode_#newpipe:matrix.org`, or is there actually a separate Matrix room I'm unaware of?

Reply:
good catch!
thanks